app/src/sharedTest/java/org/oppia/android/app/thirdparty/LicenseTextViewerFragmentTest.kt

```diff
@@ -317,6 +321,36 @@ class LicenseTextViewerFragmentTest {
     }
   }
 
+  @Test
+  fun testFragment_fragmentLoaded_verifyCorrectArgumentsPassed() {
+    launch<LicenseTextViewerActivity>(
+      createLicenseTextViewerActivity(
+        dependencyIndex = 3,
+        licenseIndex = 1
+      )
+    ).use { scenario ->
+      testCoroutineDispatchers.runCurrent()
+      scenario.onActivity { activity ->
+
+        val licenseTextViewerFragment = activity.supportFragmentManager
+          .findFragmentById(R.id.license_text_viewer_fragment_placeholder)
+          as LicenseTextViewerFragment
+        val arguments = checkNotNull(licenseTextViewerFragment.arguments) {
+          "Expected arguments to be passed to LicenseTextViewerFragment"
+        }
+        val args = arguments.getProto(
+          "LicenseTextViewerFragment.arguments",
```

Comment:
```suggestion
          LicenseTextViewerFragment.LICENSE_TEXT_VIEWER_FRAGMENT_ARGUMENTS_KEY,
```

Reply:
We need to make LICENSE_TEXT_VIEWER_FRAGMENT_ARGUMENTS_KEY public  in LicenseTextViewerFragment to use it here. 
Could you please confirm if this change is acceptable?

Reply:
Making it public should be fine.